renovate.json

```diff
@@ -29,17 +29,17 @@
         "groupName": "golang.org/x"
       },
       {
-        "matchPackageNames": ["cloud.google.com/go**"],
-        "groupName": "cloud.google.com/go"
+        "matchPackageNames": ["cloud.google.com/**"],
```

Comment:
Was removing `go` intentional ?

Reply:
yes.  It will group any update under cloud.google.com.  The previous match didn't work for cloud.google.com/go/longrunning, for example